testing/picotts/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Eric Trombly <etrombly@yahoo.com>
+# Maintainer:
+pkgname="picotts"
+pkgver="0.1"
+pkgrel=0
+pkgdesc="Pico TTS: text to speech voice synthesizer from SVox"
+url="https://github.com/naggety/picotts"
+arch="all"
+license="Apache-2.0"
+depends="popt"
+makedepends="popt-dev autoconf automake libtool"
+install=""
+subpackages="$pkgname-dev"
+source="https://github.com/naggety/picotts/archive/e3ba46009ee868911fa0b53db672a55f9cc13b1c.zip"
```

Comment:
this is not a versioned archive

Reply:
The author hasn't made any releases, so I'm grabbing a specific commit. How is this normally handled?

Reply:
Creating an issue on the repository of the author asking for a release and then using it. If he takes too long a git commit with version `0_git$(date)` is acceptable but we really want a tagged release

Reply:
You'll need to adjust the package version.

Reply:
Still no response from the author. I'll give it another day or two.